tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -169,13 +169,13 @@ def setup(rand_selected_dut, rand_unselected_dut, tbinfo, vlan_name, topo_scenar
                 upstream_port_ids.append(port_id)
     else:
         downstream_ports = list(mg_facts["minigraph_vlans"][vlan_name]["members"])
-        # Put all portchannel members into dst_ports
+        # Put all upstream ports into dst_ports
         upstream_port_ids = []
         upstream_ports = []
-        for _, v in mg_facts['minigraph_portchannels'].items():
-            for member in v['members']:
-                upstream_port_ids.append(mg_facts['minigraph_ptf_indices'][member])
-                upstream_ports.append(member)
+        for port in mg_facts['minigraph_ports']:
+            if port not in downstream_ports and port in mg_facts['minigraph_ptf_indices']:
```

Comment:
Do we have mgmt port or internal ports for T2 devices besides upstream and downstream? If that's the case, it could add non upstream ports for those platforms, could you help to check and confirm? @Pterosaur